sbin/exonset-to-seqinfo

```diff
@@ -49,6 +49,8 @@ if __name__ == "__main__":
     ac_re = re.compile("[NX][CGMPR]_")
 
     opts = parse_args(sys.argv[1:])
+    input_dir = os.path.dirname(opts.FILES[0])
```

Comment:
Can we add an output directory to the supported arguments. So we can have this file directed to the directory the user chooses?